src/main/java/com/github/rodionovsasha/commentservice/entities/User.java

```diff
@@ -0,0 +1,30 @@
+package com.github.rodionovsasha.commentservice.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.io.Serializable;
+
+@NoArgsConstructor
+@Getter @Setter
+@Entity
+public class User implements Serializable {
+    private static final long serialVersionUID = 28116440335967L;
```

Comment:
How did you generate this `serialVersionUID`?

Reply:
I choose some random number